### DISCUSSION TITLE
lights out management

### DISCUSSION THREAD

**Discussion by meta-github** [micromdm/micromdm#769] [Q&A] [answered]
My understanding for using lights out management on the Mac Pro or M1 Mac Mini (10GB/s ethernet) is:
- controller and managed system needs to be configured using a configuration profile (type: com.apple.lom)
- the MDM needs to support LOMDeviceRequestCommand and LOMSetupRequestCommand

So, I think that currently, micromdm cannot be used for this due to lack of the two new commands. Am I right here?

**Comment by jessepeterson** [comment, marked as answer]
MicroMDM added support for the LOM commands in [v1.10](https://github.com/micromdm/micromdm/releases/tag/v1.10.0) as part of #839. The v1.10 release notes have links to example documentation from Apple. As well MicroMDM [v1.11](https://github.com/micromdm/micromdm/releases/tag/v1.11.0) added (and all NanoMDM versions have had) support for submitting *raw* commands to devices (so you can submit any command you like — like the lights-out-management commands). So LOM should be supported now.

**Comment by LouisStAmour** [comment]
Can anyone speak to how LOM works for them in practice? I'd test it myself but I don't have the Apple Enterprise plan necessary to run MicroMDM and as this is the only feature I absolutely would need an MDM for, it's hard to justify signing up for it at USD 299 a year (even if I can get approved, that is, since I'm not 100 people).

Don't get me wrong, btw, having now tried a million different MDMs including the free ManageEngine plan, Jamf Now, and many others, I'm just frustrated that apparently Apple only expects enterprises to want to use LOM. They sell an entry-level priced LOM device, but they barely support it as a feature.

It's frustrating. Some wags online suggested you could get a smart outlet and just cycle it using that to wake it up. But I just tested it, and when the Mac is properly shut down, it doesn't wake itself up. If I buy even the fanciest UPS and then during a power outage, the UPS instructs the Mac to shut down, I've no way to wake it up unless I physically push the power button. 

I can't even wake the Mac when using a plugged-in Magic Keyboard with Touch ID. When it's sleeping, the button around the Touch ID wakes it up (well, technically any key on the keyboard wakes it up) but if it’s fully shut down, the only way I can get that Apple startup chime to play is if I push the button on the bottom.

I know I could work around needing to enter a password by using some kind of fake USB keyboard, like how you can push a YubiKey to automatically enter a static password when the authentication passes. But I can’t get around automating the power button push. It would be nice if there was an Apple-supported method. Is LOM the answer here?

**Reply by jessepeterson** [reply]
I'll reply more on topic later, but just wanted to point out (as was mentioned #1051) you've not needed the Enterprise developer account for some time now. A standard developer account will do.

**Reply by LouisStAmour** [reply]
Thanks very much for the rapid reply with the pointer to the MDM CSR Option as a way to avoid the Enterprise plan. I hadn't seen that anywhere yet on Reddit or the docs here so I might submit that PR later when back at my computer. No idea how formal this process is, e.g. documented by Apple, but it does make the $99/year developer account look very enticing again. 

I stopped being a developer roughly when I could get developer betas more easily with a free account and when they relaxed iOS development so that you could build and run apps on device for a limited time without an account. But I will sign up again if it's this easy, and not just for MDM... I really want to get back into iOS development too ;-)

Edit: Found documentation from Apple of this option at https://developer.apple.com/help/account/manage-your-team/requesting-access-to-vendor-signing-certificate/
